apko/translate_lock.bzl

```diff
@@ -92,7 +92,11 @@ def _translate_apko_lock_impl(rctx):
 
     for repository in lock_file["contents"]["repositories"]:
         name = util.sanitize_string("{}_{}_{}".format(target_name, repository["name"], repository["architecture"]))
-        targets.append("@{}//:index".format(name))
+
+        # Since APKO >0.12 the indexes are not needed for acting on resolved (aka locked) image.
```

Comment:
this is a breaking change and should be conditional in rule level not here.

Reply:
Now it's conditional on Apko version -> driven on the consumption side (from apko_image rule). PTAL